tests/__tests__/configurations.test.ts

```diff
@@ -207,7 +207,7 @@ describe('test initValidate resolves', () => {
   test('test with all defaults', async () => {
     const testConfig = {
       namespace: 'sp1',
-      networkConfig: {endpoint: 'test', method: 'post'},
+      networkConfig: {endpoint: 'test', method: 'post', customPostPath: '/'},
```

Comment:
Good point! I changed it to `com.snowplowanalytics.snowplow/tp2` because the leading slash is added by default. Also changed in other places to remove the leading slash.